src/seedu/addressbook/common/Utils.java

```diff
@@ -1,14 +1,16 @@
 package seedu.addressbook.common;
 
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.Set;
+import seedu.addressbook.data.person.Printable;
+
+import java.util.*;
```

Comment:
Do refer back to the coding standard which states that `Imported classes should always be listed explicitly.`